doc/development/e2e.md

```diff
@@ -157,7 +155,11 @@ cd ..
 APPIUM_APP=./PassCulture.zip yarn e2e:ios.app
 ```
 
-> It is not possible to test the iOS application using the development environment
+If the application is already installed, you can test it without building the application, for instance, for testing app you will do:
```

Comment:
```suggestion
If the application is already installed, you can inspect it (with Appium Inspector) without building the application, for instance, for testing app you will do:
```

> Since you cannot erase application cache on iOS, re-running test will fail due to FirstLaunch requirements.